jekyll/_cci2/use-the-cli-for-config-policy-management.adoc

```diff
@@ -0,0 +1,226 @@
++
+[source,shell]
+----
+circleci-cli policy push ./config --owner-id <your-organization-ID>
+----
++
+Now, when a pipeline is triggered within your organization, the project's `.circleci/config.yml` will be validated against this policy.
+
+== Update a policy
+
+To illustrate making a change to an existing policy, suppose you made an error when creating that policy, and that configs in your organization are using CircleCI config version `2.0` and that you want your policy to reflect this.
+
+. Change the rule definition in your `version.rego` file:
++
+[source,rego]
+----
+{
+    not input.version >= 2.0 # check that version is at least 2.0
+    reason := sprintf("version must be at least 2.0 but got %s", [input.version])
+}
+----
+
+. Push the policy directory containing the updated policy file using the CLI (verify the diff, and choose yes when prompted):
++
+[source,shell]
+----
+circleci-cli policy push ./config --owner-id <your-organization-ID>
+----
+
+== Manage policies with your VCS
+
+CircleCI policies are managed by pushing directories of policies to CircleCI via the CLI. We recommend managing your policy directory via your VCS by storing your policies in a repository within your organization.
+
+This is how policies are managed internally at CircleCI. Pushing policy bundles is done by triggering CircleCI pipelines.
+
+We also recommend creating a bot account for pushing policies and to use its associated CircleCI token. The context should be restricted to groups that are responsible for managing policies. See Restricted Contexts.
+
+=== Set up config policy management with your VCS
+
+. Set up repository in your VCS to manage policies.
+
+. Create a directory for your `rego` policy files, for example:
++
+[source,shell]
+----
+mkdir ./config-policies
+----
+
+- Set up a `.circleci/config.yml` file for your policies repository. This example will push policies to CircleCI on commits to `main` and show a diff of the policy bundle on commits to all other branches:
++
+[source,yaml]
+----
+version: 2.1
+
+orbs:
+  circleci-cli: circleci/circleci-cli@0.1.9
+
+workflows:
+  main-workflow:
+    jobs:
+      - diff-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              ignore: main
+      - push-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              only: main
+
+jobs:
+  diff-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Diff policy bundle
+          command: circleci policy diff ./config --owner-id $OWNER_ID
+
+  push-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Push policy bundle
+          command: circleci policy push ./config --no-prompt --owner-id $OWNER_ID
+----
+
+Let us break down the previous config:
```

Comment:
I propose to move a lot of the explanation here into comments in the config file snippet to make it easier to follow. Let me know if you object!

Reply:
@davidmparrott

Reply:
No objections!